step.sh

```diff
@@ -1,27 +1,40 @@
 #!/bin/bash
 set -e
 
-echo "Checking status of collector at $collector_url..."
-result=$(curl --silent "$collector_url/micro/all")
+echo "Checking status of collector at ${CP_BITRISE_SNOWPLOW_MICRO_COLLECTOR_URL}/micro/all"
+result=$(curl --silent "${CP_BITRISE_SNOWPLOW_MICRO_COLLECTOR_URL}/micro/all")
+
+echo result
```

Comment:
From our call, I think we still need to use the `$collector_url` input, so lets revert this bit

Reply:
Thank you 🙏 Updated